SubmittingPatches-backports.rst

```diff
@@ -234,12 +234,26 @@ the backport branch with the cherry-picks in it.
 
 The script is located at ``src/script/ceph-backport.sh`` in the ``master``
 branch. Though there might be an older version of this script in a stable
-branch, do not use it. Only use the most recent version from master.
+branch, do not use it. Only use the most recent version from master and
+best not from your active repository. That said, copy the script to a
+dedicated path outside the repository you use or use the following
+alias that will download the most recent script on every call::
+
+    alias ceph-backport="bash <(curl -Ls https://github.com/ceph/ceph/raw/master/src/script/ceph-backport.sh)"
+
+In order to update your local copy to the most recent version,
+you could do the following::
+
+    cd $dedicatedPath
+    curl -O https://github.com/ceph/ceph/raw/master/src/script/ceph-backport.sh
+    chmod +x ceph-backport.sh
+
+Hint: You could put these three lines into a script and run it via a daily cronjob.
```

Comment:
daily seems like overkill - could you drop the word "daily"? The script does not change very often.